src/DateTimePickerAndroid.android.ts

```diff
@@ -54,19 +54,22 @@ function open(props: AndroidNativeProps) {
 
   const presentPicker = async () => {
     try {
-      const dialogButtons = {
+      const dialogButtons: Record<
+        'positive' | 'neutral' | 'negative',
+        ProcessedButton
+      > = {
         positive: {
-          label: positiveButtonLabel,
+          label: positiveButtonLabel as string,
```

Comment:
please remove the `as` cast here, when you change the `ProcessedButton` to have the values optional, TS won't complain